pkg/security/probe/probe.go

```diff
@@ -955,11 +952,20 @@ func (p *Probe) isNeededForActivityDump(eventType eval.EventType) bool {
 
 // SelectProbes applies the loaded set of rules and returns a report
 // of the applied approvers for it.
-func (p *Probe) SelectProbes(rs *rules.RuleSet) error {
+func (p *Probe) SelectProbes(eventTypes []eval.EventType) error {
 	var activatedProbes []manager.ProbesSelector
 
+	contains := func(arr []eval.EventType, eventType eval.EventType) bool {
```

Comment:
Maybe we can use https://pkg.go.dev/golang.org/x/exp/slices#Contains ?

Reply:
Done